marlowe/test/Spec/Marlowe/Marlowe.hs

```diff
@@ -450,6 +241,7 @@ valueSerialization = property $
         in Just a === decoded
 
 
+-- | Test a complicated sequence of multiplications.
```

Comment:
I've started letting the static analysis run as long as it wants, and so far it looks like it does terminate even though that might take 10+ minutes in rare cases. Next sprint I'll investigate and resolve this (SCP-4267), probably timing out on success if the 5 minute mark is reached (since the CI only allows 15 minutes).